Platform/Apple/virtual/src/font/fontEngine.s

```diff
@@ -1670,7 +1640,7 @@ Wp_CfHtVt STA Flg_PsC	;set Plot Separator flag
 	BNE Wp_VtVal	;no - then go do vTab
 	LDA T1_vLo	;yes - then hTab
 ;
-	CLC 		;hTAB: get param add it to 
+;;	CLC 		;hTAB: get param add it to 
```

Comment:
Since the previous `BNE` failed, that means CMP #5 was equal, which set the carry. So why is it safe to comment out the CLC here?